content/_config/site.yml

```diff
@@ -5,7 +5,6 @@ google_analytics:
   anonymizeIp: true
   account: UA-4216293-5
 haml:
-  :ugly: true
```

Comment:
hm?

Reply:
The option was removed in https://github.com/haml/haml/pull/894